docs/topics/sb-features-and-frameworks-summary.adoc

```diff
@@ -57,7 +57,7 @@ a|
 Server Side Load Balancing
 
 a|
-Handle load increases by deploying multiple instances of a service, transparently distributing the load across instances.
+Handles load increases by deploying multiple instances of a service; It transparently distributes the load across instances.
```

Comment:
Should it really be a semicolon ";"?

Reply:
It can be.

Reply:
Is it OK then for the following word to start with capital "I"? It looks wrong to me, but maybe that's OK in Englihs.

Reply:
Ah, I overlooked that. It should indeed be lowercase.